src/main/java/ru/ayaz/User.java

```diff
@@ -0,0 +1,63 @@
+package ru.ayaz;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+public class User {
+    private String nickname;
+    private Set<String> ignoredUsersSet;
```

Comment:
&#x1F34E; **Issue**
I think you should rename your "ignoredUsersSet" to "ignoredUsers",  its clearer.